hw/xbox/dsp/dsp_dma.c

```diff
@@ -77,13 +109,21 @@ static void dsp_dma_run(DSPDMAState *s)
             s->eol = true;
         }
 
+        /* Decode control word */
+        bool unk0 = (control >> 0) & 1; /* FIXME: What does this do? */
+        bool direction = control & NODE_CONTROL_DIRECTION;
+        uint32_t unk2 = (control >> 2) & 0x3;
+        bool unk4 = (control >> 4) & 1; /* FIXME: What does this do? */
+        uint32_t buf_id = (control >> 5) & 0xf;
+        bool unk9 = (control >> 9) & 1; /* FIXME: What does this do? */
+        uint32_t format = (control >> 10) & 0x7;
+        bool unk13 = (control >> 13) & 1;
+        uint32_t step = (control >> 14) & 0x3FF;
 
-        DPRINTF("\n\n\nDMA addr %x, control %x, count %x, "
-                 "dsp_offset %x, scratch_offset %x, base %x, size %x\n\n\n",
-                addr, control, count, dsp_offset,
-                scratch_offset, scratch_base, scratch_size);
+        /* Check for unknown control settings */
+        assert(unk2 == 0x0);
+        assert(unk13 == false);
```

Comment:
I might add `assert(step == 0x001)` to catch #164.
Please don't merge yet, until I've decided.

Reply:
Can't do this. Step is used elsewhere. We'll just have to implement it in the future.